code/controllers/subsystem/starmap.dm

```diff
@@ -341,19 +339,23 @@ SUBSYSTEM_DEF(starmap)
 	ftl_drive.status_update(message)
 
 /datum/controller/subsystem/starmap/proc/ftl_sound(var/sound) //simple proc to play a sound to the crew aboard the ship, also since I want to use minor_announce for the FTL notice but that doesn't support sound
-	for(var/area/shuttle/ftl/F in world)
-		F << sound
+	for(var/area/A in GLOB.sortedAreas)
+		var/turf/T = A.contents[1]
+		if(T.z == ZLEVEL_STATION)
+			A << sound
```

Comment:
Make the loop typeless.

Also you need to be checking if it's an area on the ship, not just an area on the same z level.

Also did you even test this? Area contents aren't always turfs, though they are atoms.

Reply:
I did test it and it works.